homeassistant/components/somfy/switch.py

```diff
@@ -0,0 +1,49 @@
+"""Support for Somfy Camera Shutter."""
+from pymfy.api.devices.camera_protect import CameraProtect
+from pymfy.api.devices.category import Category
+
+from homeassistant.components.somfy import DOMAIN, SomfyEntity, DEVICES, API
```

Comment:
Intra package imports should be relative.

Reply:
```